builder/triton/driver_mock.go

```diff
@@ -17,6 +17,9 @@ type DriverMock struct {
 	DeleteMachineId  string
 	DeleteMachineErr error
 
+	GetImageId  string
```

Comment:
Style nit: `Id` -> `ID`, but the rest of the builder is using `Id`.  Maybe in the future we can make a second pass for that and modernize the style.